os/arch/arm/Kconfig

```diff
@@ -288,6 +288,17 @@ config ARCH_DABORTSTACK
 		will be used during data abort handling. Recommended data abort stack
 		size is 1K.
 
+choice
+	prompt "Stack Protection feature"
+
+config REG_STACK_OVERFLOW_PROTECTION
+	bool "REG Stack Overflow Protection"
+	depends on ARCH_ARMV8M_FAMILY
+	default n
```

Comment:
This is wrong syntax of choice.
Each config under choice should not have default.
Choice should have the default.

Reply:
How to disable stack protection? You miss the way to disable.

Reply:
changed to meet the requirement